server/models/clientModel.ts

```diff
@@ -11,8 +11,7 @@ type ClientDocument = ClientInterface & Document;
 
 const ClientSchema = new Schema({
   clientId: {
-      type: String,
-      required: true
+    type: String,
```

Comment:
How come this field is no longer required?

Reply:
this was supposed to be the ekyros id of the client. 
I don't want to put any garbage/placeholder default value, so Im just leaving it out of all client objects until we know what data we're storing on clients